src/ImageSharp/Formats/Png/PngTransparentColorBehavior.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Six Labors and contributors.
+// Licensed under the GNU Affero General Public License, Version 3.
+
+namespace SixLabors.ImageSharp.Formats.Png
+{
+    /// <summary>
+    /// Enum indicating how the transparency should be handled on encoding.
+    /// </summary>
+    public enum PngTransparentColorBehavior
```

Comment:
My bad, let's call it `PngTransparentColorMode` to keep it in line with othernaming.